chia/rpc/wallet_request_types.py

```diff
@@ -171,7 +171,18 @@ class GetHeightInfoResponse(Streamable):
 @streamable
 @dataclass(frozen=True)
 class PushTX(Streamable):
-    spend_bundle: bytes
+    spend_bundle: WalletSpendBundle
+
+    # We allow for flexibility in transaction parsing here so we need to override
+    @classmethod
+    def from_json_dict(cls, json_dict: dict[str, Any]) -> PushTX:
+        if isinstance(json_dict["spend_bundle"], str):
```

Comment:
if "spend_bundle" isn't in the json_dict this will raise - that is desired here?

Reply:
Yeah, the class is a serialization spec and the spend_bundle argument is not provided with a default which means that even without this method it would have raised (and should)